app/js/components/ticker-list.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+import Ticker from './ticker'
+
+export default class TickerList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {stocks:[]};
+  }
+
+  componentDidMount () {
+    var stockRequest = new XMLHttpRequest();
+    var symbol = this.props.symbol;
+    var stockInfo, stocks;
+    if (this.props.symbol) {
+      stockRequest.open('GET', (`./api/stocks/${symbol}`));
```

Comment:
Also, this seems like it's a little misplaced - `TickerList` implies that this is a list and not an individual Ticker